demo/index.html

```diff
@@ -16,10 +16,39 @@
 
     <demo-navigation></demo-navigation>
 
-    <h3>Sample example</h3>
+    <h3>Default button</h3>
     <demo-snippet>
       <template>
-        <vaadin-button></vaadin-button>
+        <vaadin-button>Vaadin Button</vaadin-button>
+        vs
+        <button>Native button</button>
+      </template>
+    </demo-snippet>
+
+    <h3>Button with autofocus</h3>
+    <demo-snippet>
+      <template>
+        <vaadin-button autofocus>
+          <img src="vaadin-logo-small.png">
```

Comment:
The image file is an old logo, don’t use.

Please also consider high-DPI screens support.